packages/frontend/src/components/ProjectAccess/ProjectAccess.styles.ts

```diff
@@ -26,19 +25,42 @@ export const UserName = styled.b`
     margin-right: 0.625em;
 `;
 
-export const UserEmail = styled(Tag)`
-    width: fit-content;
-    margin-top: 0.3em;
+export const ButtonGroup = styled.div`
+    display: flex;
```

Comment:
I would avoid improving/keeping Styled components here, could we do the same with moving these styles + components from `ProjectAccess.styles.ts` to `ProjectAccess.tsx` without using Styled components? Apologies if this is not clear